M2/Macaulay2/d/version.dd

```diff
@@ -206,6 +208,8 @@ setupconst("version", Expr(toHashTable(Sequence(
    "readline version" => readlineversion(),
    "mpfr version" => Ccode(constcharstar,"MPFR_VERSION_STRING"),
    "mpfi version" => Ccode(constcharstar,"MPFI_VERSION_STRING"),
+   "tbb version" => Ccode(constcharstar,
+       "stringize(TBB_VERSION_MAJOR) \".\" stringize(TBB_VERSION_MINOR)"),
```

Comment:
Maybe use this instead:
```c
// Full official version string
#define TBB_VERSION_STRING __TBB_STRING(TBB_VERSION_MAJOR) "." __TBB_STRING(TBB_VERSION_MINOR) __TBB_VERSION_SUFFIX
```

Reply:
Where is `__TBB_VERSION_SUFFIX` defined?  I don't see it in any of the headers on my system...

Reply:
It looks like it's been added recently.  It's on their master branch: https://github.com/oneapi-src/oneTBB/blob/master/include/oneapi/tbb/version.h

It looks like that would be nice to have if it's available -- I'll add an #ifdef in there.

Reply:
It's in a completely new header file starting with tbb 2021 (`version.h` instead of `tbb_stddef.h`), so this is a bit more complicated since we'd need to deal with the header file in both the autotools and cmake builds...  I think I'll hold off for now, since other parts of the build fail using tbb 2021 (#2105).